Gemfile

```
@@ -1,5 +1,7 @@
 source "https://rubygems.org"
 
+gem "webrick"
+
 group :jekyll_plugins do
     gem "github-pages"
     gem "jekyll-glossary_tooltip", "~> 1.5"
```

I needed to add this gem

```suggestion
    gem "jekyll-glossary_tooltip", "~> 1.5"
    gem "activesupport", "~> 7.0"
```

---

I'm not an experienced docker user but managed to get it to work by adding the gem "activesupport"
I also got a warning that the version attribute in docker-compose.yml is obsolete and should be removed to not cause confusion.

Also, there are already instructions regarding the use of docker on this page: pages/contribute/working_with_git.md 
I could not get those to work and nevertheless they should be harmonised.